### DISCUSSION TITLE
How to use this logo name with '&' in shields badge url?

### DISCUSSION THREAD

**Discussion by RoopanJK** [badges/shields#9037] [Q&A] [answered]
How do i add the logo name in the following URL, doing so as below doesn't seem to work

https://img.shields.io/badge/-B&R%20Automation%20Studio-FF8800?logo=b&r%20automation&style=for-the-badge

![image](https://user-images.githubusercontent.com/82260000/228765693-76d3bd13-88b0-4388-9e6f-84bb01e9990a.png)

**Comment by chris48s** [comment, marked as answer]
The logo slug for this is `bandrautomation`: https://simpleicons.org/icons/bandrautomation.svg
So you would use https://img.shields.io/badge/-B&R%20Automation%20Studio-FF8800?logo=bandrautomation&style=for-the-badge
but you'll probably actually want to set the logo colour to something visible e.g: https://img.shields.io/badge/-B&R%20Automation%20Studio-FF8800?logo=bandrautomation&style=for-the-badge&logoColor=white

**Reply by LitoMore** [reply]
Hi @RoopanJK, you can click the icon title to copy the slug.

<img width="237" alt="CleanShot 2023-04-07 at 15 56 57@2x" src="https://user-images.githubusercontent.com/8186898/230567719-1d8503e5-9d92-4236-8186-641910ddce50.png">

I also created a PR to add a hint for this: #9054

**Reply by RoopanJK** [reply]
Thanks a lot!
